### PR TITLE
Feature/homepage

### PR DESCRIPTION
**Pull request by cliftondavies**
# Project: Using Bootstrap
## Milestones
- [x] Setup files
- [x] A fully responsive replica of [Newsweek](https://www.newsweek.com/)
- [x] HTML and CSS Validator
- [x] Local Stylelint
- [x] GitHub Stickler
- [x] README

### CODE REVIEW

**Comment by majovanilla** [comment]
### Code under review by Maria Reyes

**Comment by majovanilla** [comment]
Hey @cliftondavies  👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯

- Good Readme file
- Good Pull Request Message
- Great use of Bootstrap classes

I really like your clone!! So don't worry if you see a lot of requested changes, your page looks wonderful but there's more work to do.

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to check your HTML using the [validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fgithub.com%2Fcliftondavies%2FNewsweek%2Fblob%2Ffeature%2Fhomepage%2Findex.html). You have some errors and warnings. Please fix them.

- Please check the comment I made on your code and implement the changes.

**Changes on small screen size**
- The order of the sections is not correct. Check how the main section is behaving in the original page and edit your file. 
Newsweek page: 
1-Top Story 2- Featured Stories 3- Opinion 4-More stories 5- Culture and travel 6- Latest news
Screenshot from 2020-02-21 20-06-18

your page:
1-Top Story 2- More Stories 3- Culture and travel 4-Featured stories 5- Opinion 6- Latest news
Screenshot from 2020-02-21 20-06-10

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes



You can find me on Social Media below 👇 . Remember to say hi!

[GitHub](https://github.com/majovanilla) [Twitter](https://twitter.com/MajoVanilla) [LinkedIn](https://www.linkedin.com/in/majoreyesparroquin/)

File: assets/styles/style.css [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+
+/*
+ * Highlights Section
+ *
+ * Includes subsections: Featured, Middle and Right.
+ *
+ * Middle sub-sections: Top Story, More Stories, and Culture & Travel.
+ *
+ * Right sub-sections: Latest News, and Opinion.
+ */
```
Comment (majovanilla) [inline assets/styles/style.css:202]: Please delete the commented code.

**Comment by cliftondavies** [comment]
:wave: @majovanilla 

Thanks for the prompt, and detailed review. :pray::pray:

## Response to required changes:

### Always remember to check your HTML using the validator. You have some errors and warnings. Please fix them.

I checked both  HTML and CSS prior to opening a pull request. There were no errors in the **index.html**, and had only one warning in my **style.css** file about `background-color` and `border-color` being the same for the `.second-navbar::before` selector. I just checked again on your request, and got the same result. :grinning: 

Important to note that the **W3C CSS Validator** will include the **bootstrap.min.css** file from the **bootstrap cdn** if you validate by URL. The errors and warnings you are referring to are within that remote css file, at the absolute location, https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css.

### Please delete the commented code

**Line 191-199** isn't commented code. It is a **multi-line CSS comment** used to provide further description for sections within css files. It is recommended for better organisation of css styling. Please see [Code Guide suggestions on organisation of css](https://codeguide.co/#css-organization) by Twitter's Mark Otto, for reference.

 ### Changes on small screen size

The order you see in my demo is how it was on small screens up until the time I opened a pull request. I tripled check the order on different screen sizes before submitting. I noticed the order for small screens has changed overnight, literally, but you can check recent submissions for this project from other students to confirm this.


Hope this helps to clarify a few things.

Thanks again, and best wishes :100: :1st_place_medal:

@cliftondavies

**Review by MarioDena — CHANGES_REQUESTED** [review]
## Hello Clifton Davies! My name is Mario Enrique Delgado Naranjo

After reading your comments I have checked your project again. You really did a great job, :+1: however we need some changes.

About previously requested changes, I checked your project using https://web.archive.org/ on date Feb-07 and you are right your layout is correct. I apologize for the inconvenient. Keep this link handy for future reference.

You HTML file is indeed __valid__ I'm sorry about the false negative on the last review.

## Changes Required:

- [ ] Please remove all comments on your code Both on your HTML and CSS files. I know you are following a good structure, however think of the requirements for your projects as mandatory from a client. Instead of using comments the program encourages you to use descriptive names for your classes and ids on CSS and semantic HTML on HTML. (You are already doing that) 

## Optional Changes: 

- [ ] Your Newsweek Logo is not centered, it's tilted to the left this is caused by other items and their width difference. You can try to find out how to pixel perfect center your logo. _This change is `[optional]`_

- [ ] All your social icons change to the same color on hover. Try giving each social Icon it's own color matching that site color-scheme.  _This change is `[optional]`_

- [ ] Try to add an independent horizontal navigation to your center row on medium and big screens like in the original, and hide the scrollbar.  _This change is `[optional]`_

After making the requested changes please resubmit your project.

Again I congratulate you on your project it's one of the most accurate clones I've seen. 

Cheers! :smiley:

**Comment by cliftondavies** [comment]
Thanks @MarioDena for your review, and optional suggestions. :+1: 

All code that was commented out has been removed. I also took up the option to center the header logo. :grinning:

**Comment by uryelah** [comment]
Hello **Clifton Davies**, I'm Sarah, your last reviewer for this PR  😄 .

You did an amazing job completing the required changes 🥇 

----

### Your project is approved! 🎊 🥅 💛 
### 

**You may close this Pull Request and merge the changes**

-------

**Keep being awesome devs and good luck with your next project!**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by cliftondavies** [comment]
Thanks **Sarah**  👍 🙏

**Pull request merged**